core/src/main/java/com/scalar/db/transaction/jdbc/JdbcTransactionAdmin.java

```diff
@@ -153,4 +156,56 @@ public void upgrade(Map<String, String> options) throws ExecutionException {
   public void close() {
     jdbcAdmin.close();
   }
+
+  @Override
+  public void createUser(String username, @Nullable String password, UserOption... userOption) {
+    throw new UnsupportedOperationException();
```

Comment:
Initially, I didn't add any message here because we were going to remove the JDBC transaction and ScalarDB Server components. However, recent discussions have raised the possibility of retaining the JDBC transaction. Therefore, I will add a message depending on the outcome of this discussion:
https://github.com/scalar-labs/scalardb/pull/1318#discussion_r1403852190